liota/lib/transports/mqtt.py

```diff
@@ -271,6 +280,7 @@ def connect_soc(self):
         self._paho_client.connect(host=self.url, port=self.port, keepalive=self.keep_alive)
 
         # Start network loop to handle auto-reconnect
+        self._paho_client.reconnect_delay_set(random.randint(1, 15), random.randint(150, 250))
```

Comment:
Why these numbers where chosen? How many clients did we try out with? Are we sure this will succeed for thousands of client?
We should make these parameters user configurable so that QE can test and set it to correct values, instead of some randomly chosen numbers.